.tutone.yml

```diff
@@ -544,6 +544,36 @@ packages:
         field_type_override: nrtime.DateTime
         skip_type_create: true
 
+  - name: droprules
+    path: pkg/droprules
+    import_path: github.com/newrelic/newrelic-client-go/pkg/droprules
+    generators:
+      - typegen
```

Comment:
Check out [this PR](https://github.com/newrelic/newrelic-client-go/pull/665/files#diff-c8fb5f06f26ca2ea1554da789ba1914c18e334991add8cbd66482c4568907a17R598) for auto-generating the queries / mutations as well (we're trying to do that as much as possible)